keras_cv/models/segmentation/segment_anything/sam_test.py

```diff
@@ -220,48 +220,68 @@ def test_mask_decoder(self):
         self.assertEqual(num_parameters, 4_058_340)
 
     @pytest.mark.large
-    def test_end_to_end_model_predict(self):
-        model = SegmentAnythingModel(
-            backbone=self.image_encoder,
-            prompt_encoder=self.prompt_encoder,
-            mask_decoder=self.mask_decoder,
-        )
-
-        # We use box-only prompting for this test.
-        mask_prompts = self.get_prompts(1, "boxes")
-        inputs = {
-            "images": np.ones((1, 1024, 1024, 3)),
-        }
-        inputs.update(mask_prompts)
-
-        # Check the number of parameters
-        num_parameters = np.sum([np.prod(x.shape) for x in model.weights])
-        self.assertEqual(num_parameters, 89_670_912 + 6_476 + 4_058_340)
+    @parameterized.named_parameters(
```

Comment:
why is this test marked as large? (just for my own learning)

Reply:
The model initialized here is a ViT Base model with 130M parameters. Creating and evaluating it takes about 15-20 seconds which is significantly more than small unit tests in KerasCV.

Reply:
Gotcha, thanks! No need on this PR, but in general it will be good to separate small checks (like dtype stuff) into fast running tests, and keep the large test only for the things that must inherently by large parameter count and slow (like preset tests).

Did a big rewrite of KerasNLP backbones to this effect a bit ago. e.g. https://github.com/keras-team/keras-nlp/blob/a05f411a27eab437e71a1651c97e9addf26298ef/keras_nlp/models/bert/bert_backbone_test.py#L38-L80